cmake/BuildOptionDefaults.cmake

```diff
@@ -12,6 +12,7 @@ set(NRN_ENABLE_DISCRETE_EVENT_OBSERVER_DEFAULT ON)
 set(NRN_ENABLE_PYTHON_DEFAULT ON)
 set(NRN_ENABLE_THREADS_DEFAULT ON)
 set(NRN_ENABLE_MPI_DEFAULT ON)
+set(NRN_ENABLE_MUSIC_DEFAULT ON)
```

Comment:
Seems like the default should be OFF

Reply:
I agree - wanted to ask you the same before I change it to OFF.

Reply:
It should be OFF. The "ON" build can't succeed unless MUSIC is installed. If it's ok to build the wheels with libmusic.a (with ```-fPIC```) so that it literally gets incorporated into libnrniv.so, then we can explicitly add the option (and build MUSIC) to the wheel build scripts (perhaps as a separate PR). I can do the same for the mac pkg build script. @mdjurfeldt When we get around to needing libmusic.a with ```-fPIC``` perhaps you can help us with that option for the MUSIC configure.